runtime/cmd/main.go

```diff
@@ -61,6 +61,7 @@ func main() {
 		ConnectionCacheSize: 100,
 		MetastoreDriver:     conf.DatabaseDriver,
 		MetastoreDSN:        conf.DatabaseURL,
+		QueryCacheSize:      1000,
```

Comment:
We should add options for `ConnectionCacheSize` and `QueryCacheSize` in `conf` (which is populated through env vars)

Reply:
done